java/ql/lib/semmle/code/java/frameworks/apache/Collections.qll

```diff
@@ -182,6 +204,695 @@ private class ApacheKeyValueModel extends SummaryModelCsv {
+          ".list;AbstractLinkedList$Node;true;setPreviousNode;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;AbstractLinkedList$Node;true;getNextNode;;;Element of Argument[-1];Element of ReturnValue;value",
+          ".list;AbstractLinkedList$Node;true;setNextNode;;;Element of Argument[-1];Element of Argument[0];value",
+          ".list;AbstractLinkedList$Node;true;setNextNode;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;AbstractListDecorator;true;AbstractListDecorator;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;AbstractSerializableListDecorator;true;AbstractSerializableListDecorator;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;CursorableLinkedList;true;CursorableLinkedList;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;CursorableLinkedList;true;cursor;;;Element of Argument[-1];Element of ReturnValue;value",
+          ".list;CursorableLinkedList$Cursor;true;Cursor;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;CursorableLinkedList$SubCursor;true;SubCursor;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;FixedSizeList;true;FixedSizeList;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;FixedSizeList;true;fixedSizeList;;;Element of Argument[0];Element of ReturnValue;value",
+          ".list;GrowthList;true;GrowthList;(List);;Element of Argument[0];Element of Argument[-1];value",
+          ".list;GrowthList;true;growthList;;;Element of Argument[0];Element of ReturnValue;value",
+          ".list;LazyList;true;LazyList;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;LazyList;true;lazyList;;;Element of Argument[0];Element of ReturnValue;value",
+          ".list;NodeCachingLinkedList;true;NodeCachingLinkedList;(Collection);;Element of Argument[0];Element of Argument[-1];value",
+          ".list;PredicatedList;true;PredicatedList;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;PredicatedList;true;predicatedList;;;Element of Argument[0];Element of ReturnValue;value",
+          ".list;SetUniqueList;true;SetUniqueList;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;SetUniqueList;true;SetUniqueList;;;Element of Argument[1];Element of Argument[-1];value",
+          ".list;SetUniqueList;true;setUniqueList;;;Element of Argument[0];Element of ReturnValue;value",
+          ".list;SetUniqueList;true;asSet;;;Element of Argument[-1];Element of ReturnValue;value",
+          ".list;SetUniqueList;true;createSetBasedOnList;;;Element of Argument[1];Element of ReturnValue;value",
+          ".list;TransformedList;true;TransformedList;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;TransformedList;true;transformingList;;;Element of Argument[0];Element of ReturnValue;value",
```

Comment:
missing `transformedList`

Reply:
The docs say "If there are any elements already in the list being decorated, they will be transformed by this method." So this can't be properly modelled until we have lambda flow.